app/views/tomatoes/_form.html.erb

```diff
@@ -1,21 +1,8 @@
 <% remote = false if local_assigns[:remote].nil? %>
 
-<%= form_for(@tomato, remote: remote) do |f| %>
-  <% if @tomato.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@tomato.errors.count, "error") %> prohibited this tomato from being saved:</h2>
+<%= bootstrap_form_for(@tomato, remote: remote) do |f| %>
+  <%= render 'shared/form_errors', model: @tomato %>
 
-      <ul>
-      <% @tomato.errors.full_messages.each do |msg| %>
-        <li><%= msg %></li>
-      <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-  <div class="form-group">
-    <%= f.label :tag_list %>
-    <%= f.text_field :tag_list, class: 'form-control input-lg', placeholder: 'cook, dessert' %>
-  </div>
-  <%= f.submit class: 'btn btn-default', :'data-disable-with' => t('saving') %>
+  <%= f.text_field :tag_list, class: 'form-control input-lg', placeholder: 'cook, dessert' %>
+  <%= f.submit :'data-disable-with' => t('saving') %>
```

Comment:
There is a little regression on the submit button (see the image below). Could you fix it?

![screenshot from 2017-09-20 09-34-03](https://user-images.githubusercontent.com/56606/30632192-8fd60c24-9de7-11e7-8533-a7cde2fa3ab2.png)

Reply:
Another great catch! 💯  for the screenshot!

`submit` should include those classes by default, see https://github.com/bootstrap-ruby/rails-bootstrap-forms/blob/0fac4b176e753e2883b6632ab62926981cc88550/lib/bootstrap_form/helpers/bootstrap.rb#L5, do you have any idea why this doesn't render as expected @madeindjs?

Reply:
I think that Rails Bootstrap Forms does not support `data-disable-with` attribute. I will try use simple [`submit_tag`](http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-submit_tag) method

Reply:
I don't see regression but I changed submit button and I used [`submit_tag`](http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-submit_tag) method on https://github.com/madeindjs/tomatoes/commit/5ccb329aabd1e35c7d6e13a3dcb1f4f73c3cf3fe. Is this commit fix your issue?

Reply:
Can you run the app and take a screenshot @madeindjs please?

Reply:
> I think that Rails Bootstrap Forms does not support `data-disable-with` attribute

I don't think so, it should be just proxying options to a `submit_form` call, but I'd be happy to be wrong.

Reply:
There is (in french) ![tag](https://user-images.githubusercontent.com/11815139/30980047-eb73ecba-a47f-11e7-9062-99959b6618b4.png)

Reply:
@dalpo can you take one last look to this PR before approval?